pkg/network/tracer/tracer_linux_test.go

```diff
@@ -992,23 +992,27 @@ func (s *TracerSuite) TestDNATIntraHostIntegration() {
 	require.NoError(t, err, "error connecting to client")
 	defer conn.Close()
 
-	_, err = conn.Write([]byte("ping"))
-	require.NoError(t, err, "error writing in client")
+	var incoming, outgoing *network.ConnectionStats
+	require.Eventually(t, func() bool {
```

Comment:
consider using EventuallyWithT, it's can be a lot easier to follow

Reply:
Thanks, done